tsdb/meta.go

```diff
@@ -1311,6 +1291,7 @@ func (a Measurements) union(other Measurements) Measurements {
 
 // Series belong to a Measurement and represent unique time series in a database
 type Series struct {
+	mu   sync.RWMutex
```

Comment:
Ah the space between `Tags` and `id` threw me. Can you close that up if `mu` is protecting everything?